src/Text/LLVM/PP.hs

```diff
@@ -678,7 +648,7 @@ ppLoad ty ptr mo ma =
   where
   isAtomic = isJust mo
 
-  isImplicit = checkConfig cfgLoadImplicitType
+  isImplicit = llvmVer < llvmV3_7
```

Comment:
This is a nit pick, but now that we are explicitly checking version numbers inline like this, I think I'd prefer phrasing this code like so:

```hs
ppLoad ty ptr mo ma =
  "load" <+> (if isAtomic   then "atomic" else empty)
         <+> (if isExplicit then explicit else empty)
         <+> ...
  where
  ...
  isExplicit = llvmVer >= llvmV3_7
```

This has a couple of benefits:

1. I prefer to check for the most recent version first so that if we ever have to refine this check further, it will read more straightforwardly:

   ```hs
     isExplicit
       | llvmVer >= 18
       = ...
       | llvmVer >= 17
       = ...
       | llvmVer >= llvmV3_7
       = ...
       ...
   ```
2. There is some added symmetry with the `isAtomic` check above.